docs/09-faq.md

```diff
@@ -9,11 +9,11 @@ title: FAQ
 
 <details>
   <summary>How much does Up Pro cost?</summary>
-  <p>Up's subscription fee is currently $20/mo USD. When subscribed your team has access to Up Pro updates until the subscription is cancelled. When cancelled you receive access until the end if your billing period.</p>
+  <p>Up's subscription fee is currently $20/mo USD. When subscribed, your team has access to Up Pro updates until the subscription is cancelled. When cancelled, you receive access until the end of your billing period.</p>
```

Comment:
This line -- really `end if your billing period` -- was the motivation for the edits. Take as you will -- happy to pare down the edits (just edited some as I saw them).